sessions/js-dom-and-events/simple-calculator/README.md

```diff
@@ -1,7 +1,18 @@
-# simple-calculator
+# JS DOM and Events: Simple Calculator
+
+In this challenge, you’ll use JavaScript to add interactivity to a simple calculator. You’ll work with `document.querySelector` and `addEventListener` to perform calculations when buttons are clicked.
 
 ## Task
```

Comment:
```suggestion
## Tasks
```

Reply:
I actually noticed just now, that this is not consistent in our challenges - sometimes we use "Tasks" or "Task". 🙁 I can change it everywhere to "Tasks" as it would make more sense. Right?

Reply:
I replaced "Task" everywhere with "Tasks".